sound/soc/intel/boards/Kconfig

```diff
@@ -511,6 +511,20 @@ config SND_SOC_INTEL_SOF_PCM512x_MACH
 	  Say Y or m if you have such a device.
 	  If unsure select "N".
 
+config SND_SOC_INTEL_SOF_ES8336_MACH
+	tristate "SOF with ES8336 codec in I2S mode"
+	depends on I2C && ACPI
+	depends on ((SND_HDA_CODEC_HDMI && SND_SOC_SOF_HDA_AUDIO_CODEC) &&\
+		    (MFD_INTEL_LPSS || COMPILE_TEST))
+	select SND_SOC_ES8316
+	select SND_SOC_DMIC
+	select SND_SOC_HDAC_HDMI
```

Comment:
You can drop this. See SND_SOC_INTEL_EHL_RT5660_MACH for a simple reference for HDMI dependencies.